### ISSUE TITLE
Upload fail

### ISSUE THREAD

**Issue by ArrowBaZ** [avocode/avocode#1330]
Hello,

I upload a PSD by drag'n'drop file on projet, with 2 tabs open.
Same upload pb when drag to project in project manager view

Note upload work, still upload for few percent
Upload by website work fine

-----
App v. `2.13.0`
Avocode UID: 102722
```
TypeError: toISOString is not a function
    at String.toJSON (<anonymous>)
    at Object.<anonymous> (/opt/avocode/resources/app.asar/node_modules/core-js/modules/_ctx.js:18:15)
    at JSON.stringify (<anonymous>)
    at stringify (/opt/avocode/resources/app.asar/node_modules/json-stringify-safe/stringify.js:5:15)
    at Raven._makeRequest (/opt/avocode/resources/app.asar/node_modules/raven-js/src/raven.js:1559:22)
    at Raven._sendProcessedPayload (/opt/avocode/resources/app.asar/node_modules/raven-js/src/raven.js:1485:56)
    at Raven._send (/opt/avocode/resources/app.asar/node_modules/raven-js/src/raven.js:1442:14)
    at Raven._processException (/opt/avocode/resources/app.asar/node_modules/raven-js/src/raven.js:1290:14)
    at Raven._handleStackInfo (/opt/avocode/resources/app.asar/node_modules/raven-js/src/raven.js:1196:14)
    at Raven.captureException (/opt/avocode/resources/app.asar/node_modules/raven-js/src/raven.js:350:18)
    at e.captureException (file:///opt/avocode/resources/app.asar/dist/avocode.min.js:41:10164)
    at file:///opt/avocode/resources/app.asar/dist/avocode.min.js:7:21853
```

-----
App v. `2.13.0`
Avocode UID: 102722
```
Error: Requested index 0 is outside of range 0..0
    at CellSizeAndPositionManager.getSizeAndPositionOfCell (/opt/avocode/resources/app.asar/node_modules/react-virtualized/dist/commonjs/Grid/utils/CellSizeAndPositionManager.js:87:15)
    at CellSizeAndPositionManager._binarySearch (/opt/avocode/resources/app.asar/node_modules/react-virtualized/dist/commonjs/Grid/utils/CellSizeAndPositionManager.js:261:30)
    at CellSizeAndPositionManager._findNearestCell (/opt/avocode/resources/app.asar/node_modules/react-virtualized/dist/commonjs/Grid/utils/CellSizeAndPositionManager.js:319:21)
    at CellSizeAndPositionManager.getVisibleCellRange (/opt/avocode/resources/app.asar/node_modules/react-virtualized/dist/commonjs/Grid/utils/CellSizeAndPositionManager.js:219:24)
    at ScalingCellSizeAndPositionManager.getVisibleCellRange (/opt/avocode/resources/app.asar/node_modules/react-virtualized/dist/commonjs/Grid/utils/ScalingCellSizeAndPositionManager.js:154:47)
    at Grid._calculateChildrenToRender (/opt/avocode/resources/app.asar/node_modules/react-virtualized/dist/commonjs/Grid/Grid.js:616:65)
    at Grid.componentWillUpdate (/opt/avocode/resources/app.asar/node_modules/react-virtualized/dist/commonjs/Grid/Grid.js:503:12)
    at ReactCompositeComponentWrapper._performComponentUpdate (/opt/avocode/resources/app.asar/node_modules/react-dom/lib/ReactCompositeComponent.js:714:14)
    at ReactCompositeComponentWrapper.updateComponent (/opt/avocode/resources/app.asar/node_modules/react-dom/lib/ReactCompositeComponent.js:645:12)
    at ReactCompositeComponentWrapper.performUpdateIfNecessary (/opt/avocode/resources/app.asar/node_modules/react-dom/lib/ReactCompositeComponent.js:561:12)
    at Object.performUpdateIfNecessary (/opt/avocode/resources/app.asar/node_modules/react-dom/lib/ReactReconciler.js:157:22)
    at runBatchedUpdates (/opt/avocode/resources/app.asar/node_modules/react-dom/lib/ReactUpdates.js:150:21)
    at ReactReconcileTransaction.perform (/opt/avocode/resources/app.asar/node_modules/react-dom/lib/Transaction.js:140:20)
    at ReactUpdatesFlushTransaction.perform (/opt/avocode/resources/app.asar/node_modules/react-dom/lib/Transaction.js:140:20)
    at ReactUpdatesFlushTransaction.perform (/opt/avocode/resources/app.asar/node_modules/react-dom/lib/ReactUpdates.js:89:32)
    at flushBatchedUpdates (/opt/avocode/resources/app.asar/node_modules/react-dom/lib/ReactUpdates.js:172:19)
    at ReactUpdatesFlushTransaction.close (/opt/avocode/resources/app.asar/node_modules/react-dom/lib/ReactUpdates.js:47:7)
    at ReactUpdatesFlushTransaction.closeAll (/opt/avocode/resources/app.asar/node_modules/react-dom/lib/Transaction.js:206:25)
    at ReactUpdatesFlushTransaction.perform (/opt/avocode/resources/app.asar/node_modules/react-dom/lib/Transaction.js:153:16)
    at ReactUpdatesFlushTransaction.perform (/opt/avocode/resources/app.asar/node_modules/react-dom/lib/ReactUpdates.js:89:32)
    at Object.flushBatchedUpdates (/opt/avocode/resources/app.asar/node_modules/react-dom/lib/ReactUpdates.js:172:19)
    at ReactDefaultBatchingStrategyTransaction.closeAll (/opt/avocode/resources/app.asar/node_modules/react-dom/lib/Transaction.js:206:25)
    at ReactDefaultBatchingStrategyTransaction.perform (/opt/avocode/resources/app.asar/node_modules/react-dom/lib/Transaction.js:153:16)
    at Object.batchedUpdates (/opt/avocode/resources/app.asar/node_modules/react-dom/lib/ReactDefaultBatchingStrategy.js:62:26)
    at Object.enqueueUpdate (/opt/avocode/resources/app.asar/node_modules/react-dom/lib/ReactUpdates.js:200:22)
    at enqueueUpdate (/opt/avocode/resources/app.asar/node_modules/react-dom/lib/ReactUpdateQueue.js:24:16)
    at Object.enqueueSetState (/opt/avocode/resources/app.asar/node_modules/react-dom/lib/ReactUpdateQueue.js:209:5)
    at t.ReactComponent.setState (/opt/avocode/resources/app.asar/node_modules/react/lib/ReactComponent.js:63:16)
    at t.r._handleUserDataNotificationStateChange (file:///opt/avocode/resources/app.asar/dist/avocode.min.js:20:7764)
    at emitNone (events.js:91:20)
    at t.emit (events.js:185:7)
    at t.emitChange (/opt/avocode/resources/app.asar/node_modules/@avocode/flux/lib/store.js:47:12)
    at n._handleReceiveDataAction (file:///opt/avocode/resources/app.asar/dist/avocode.min.js:28:23022)
    at Object.ID_11 (/opt/avocode/resources/app.asar/node_modules/@avocode/flux/lib/utils.js:11:14)
    at Dispatcher._invokeCallback (/opt/avocode/resources/app.asar/node_modules/flux/lib/Dispatcher.js:198:24)
    at Dispatcher.dispatch (/opt/avocode/resources/app.asar/node_modules/flux/lib/Dispatcher.js:174:14)
    at Dispatcher.dispatch (/opt/avocode/resources/app.asar/node_modules/@avocode/flux/lib/dispatcher.js:76:28)
    at t.dispatch (/opt/avocode/resources/app.asar/node_modules/@avocode/flux/lib/action-creator.js:37:24)
    at t.value (file:///opt/avocode/resources/app.asar/dist/avocode.min.js:18:1492)
    at e.<anonymous> (file:///opt/avocode/resources/app.asar/dist/avocode.min.js:18:6698)
    at tryCatch (/opt/avocode/resources/app.asar/node_modules/regenerator-runtime/runtime.js:64:40)
    at Generator.invoke [as _invoke] (/opt/avocode/resources/app.asar/node_modules/regenerator-runtime/runtime.js:355:22)
    at Generator.prototype.(anonymous function) [as next] (/opt/avocode/resources/app.asar/node_modules/regenerator-runtime/runtime.js:116:21)
    at r (file:///opt/avocode/resources/app.asar/dist/avocode.min.js:18:4902)
    at file:///opt/avocode/resources/app.asar/dist/avocode.min.js:18:5002
    at process._tickCallback (internal/process/next_tick.js:103:7)
```

IMPORTANT: Please try to describe how did it happen.

-----
App v. `2.13.0`
Avocode UID: 102722
```
TypeError: Cannot read property 'offset' of undefined
    at CellSizeAndPositionManager.getVisibleCellRange (/opt/avocode/resources/app.asar/node_modules/react-virtualized/dist/commonjs/Grid/utils/CellSizeAndPositionManager.js:222:21)
    at ScalingCellSizeAndPositionManager.getVisibleCellRange (/opt/avocode/resources/app.asar/node_modules/react-virtualized/dist/commonjs/Grid/utils/ScalingCellSizeAndPositionManager.js:154:47)
    at Grid._calculateChildrenToRender (/opt/avocode/resources/app.asar/node_modules/react-virtualized/dist/commonjs/Grid/Grid.js:616:65)
    at Grid.componentWillUpdate (/opt/avocode/resources/app.asar/node_modules/react-virtualized/dist/commonjs/Grid/Grid.js:503:12)
    at ReactCompositeComponentWrapper._performComponentUpdate (/opt/avocode/resources/app.asar/node_modules/react-dom/lib/ReactCompositeComponent.js:714:14)
    at ReactCompositeComponentWrapper.updateComponent (/opt/avocode/resources/app.asar/node_modules/react-dom/lib/ReactCompositeComponent.js:645:12)
    at ReactCompositeComponentWrapper.performUpdateIfNecessary (/opt/avocode/resources/app.asar/node_modules/react-dom/lib/ReactCompositeComponent.js:561:12)
    at Object.performUpdateIfNecessary (/opt/avocode/resources/app.asar/node_modules/react-dom/lib/ReactReconciler.js:157:22)
    at runBatchedUpdates (/opt/avocode/resources/app.asar/node_modules/react-dom/lib/ReactUpdates.js:150:21)
    at ReactReconcileTransaction.perform (/opt/avocode/resources/app.asar/node_modules/react-dom/lib/Transaction.js:140:20)
    at ReactUpdatesFlushTransaction.perform (/opt/avocode/resources/app.asar/node_modules/react-dom/lib/Transaction.js:140:20)
    at ReactUpdatesFlushTransaction.perform (/opt/avocode/resources/app.asar/node_modules/react-dom/lib/ReactUpdates.js:89:32)
    at flushBatchedUpdates (/opt/avocode/resources/app.asar/node_modules/react-dom/lib/ReactUpdates.js:172:19)
    at ReactUpdatesFlushTransaction.close (/opt/avocode/resources/app.asar/node_modules/react-dom/lib/ReactUpdates.js:47:7)
    at ReactUpdatesFlushTransaction.closeAll (/opt/avocode/resources/app.asar/node_modules/react-dom/lib/Transaction.js:206:25)
    at ReactUpdatesFlushTransaction.perform (/opt/avocode/resources/app.asar/node_modules/react-dom/lib/Transaction.js:153:16)
    at ReactUpdatesFlushTransaction.perform (/opt/avocode/resources/app.asar/node_modules/react-dom/lib/ReactUpdates.js:89:32)
    at Object.flushBatchedUpdates (/opt/avocode/resources/app.asar/node_modules/react-dom/lib/ReactUpdates.js:172:19)
    at ReactDefaultBatchingStrategyTransaction.closeAll (/opt/avocode/resources/app.asar/node_modules/react-dom/lib/Transaction.js:206:25)
    at ReactDefaultBatchingStrategyTransaction.perform (/opt/avocode/resources/app.asar/node_modules/react-dom/lib/Transaction.js:153:16)
    at Object.batchedUpdates (/opt/avocode/resources/app.asar/node_modules/react-dom/lib/ReactDefaultBatchingStrategy.js:62:26)
    at Object.enqueueUpdate (/opt/avocode/resources/app.asar/node_modules/react-dom/lib/ReactUpdates.js:200:22)
    at enqueueUpdate (/opt/avocode/resources/app.asar/node_modules/react-dom/lib/ReactUpdateQueue.js:24:16)
    at Object.enqueueSetState (/opt/avocode/resources/app.asar/node_modules/react-dom/lib/ReactUpdateQueue.js:209:5)
    at t.ReactComponent.setState (/opt/avocode/resources/app.asar/node_modules/react/lib/ReactComponent.js:63:16)
    at t.r._handleUserDataNotificationStateChange (file:///opt/avocode/resources/app.asar/dist/avocode.min.js:20:7764)
    at emitNone (events.js:91:20)
    at t.emit (events.js:185:7)
    at t.emitChange (/opt/avocode/resources/app.asar/node_modules/@avocode/flux/lib/store.js:47:12)
    at n._handleReceivePageAction (file:///opt/avocode/resources/app.asar/dist/avocode.min.js:28:23286)
    at Object.ID_11 (/opt/avocode/resources/app.asar/node_modules/@avocode/flux/lib/utils.js:11:14)
    at Dispatcher._invokeCallback (/opt/avocode/resources/app.asar/node_modules/flux/lib/Dispatcher.js:198:24)
    at Dispatcher.dispatch (/opt/avocode/resources/app.asar/node_modules/flux/lib/Dispatcher.js:174:14)
    at Dispatcher.dispatch (/opt/avocode/resources/app.asar/node_modules/@avocode/flux/lib/dispatcher.js:76:28)
    at t.dispatch (/opt/avocode/resources/app.asar/node_modules/@avocode/flux/lib/action-creator.js:37:24)
    at t.value (file:///opt/avocode/resources/app.asar/dist/avocode.min.js:18:1638)
    at e.<anonymous> (file:///opt/avocode/resources/app.asar/dist/avocode.min.js:18:6634)
    at tryCatch (/opt/avocode/resources/app.asar/node_modules/regenerator-runtime/runtime.js:64:40)
    at Generator.invoke [as _invoke] (/opt/avocode/resources/app.asar/node_modules/regenerator-runtime/runtime.js:355:22)
    at Generator.prototype.(anonymous function) [as next] (/opt/avocode/resources/app.asar/node_modules/regenerator-runtime/runtime.js:116:21)
    at r (file:///opt/avocode/resources/app.asar/dist/avocode.min.js:18:4902)
    at file:///opt/avocode/resources/app.asar/dist/avocode.min.js:18:5002
    at process._tickCallback (internal/process/next_tick.js:103:7)
```

**Comment by vojtatranta** [comment]
Hello,
thank you for this report I am examining it today. I will give you more info later today.

**Comment by vojtatranta** [comment]
@ArrowBaZ I can't reproduce this error.

Is it happening regularly again and again? Does it work with single tab opened?

**Comment by petrbrzek** [comment]
Fixed in 2.13.1
